rpms/awips2.upc/Installer.ldm/patch/etc/ldmd.goesr

```diff
@@ -0,0 +1,161 @@
+# RE to take MUCH, MUCH LONGER TO EXECUTE (by 3 orders of magnitude in some
+# cases).  Therefore, DO NOT USE RE-s like
+#
+#     ".*UFPBA"
+#
+# Instead, simply use
+#
+#     "UFPBA"
+#
+# The regex(1) utility of this package can be used to examine the efficiency
+# of an RE.
+#
+###############################################################################
+# Include Statements
+###############################################################################
+#
+# It's possible to include other files by uncommented-out statements
+# like the following:
+#
+#INCLUDE pathname
+#INCLUDE "pathname with spaces"
+#
+# Non-absolute pathnames are relative to the directory that contains the
+# configuration-file with the include statement.
+#
+###############################################################################
+# Exec Entries
+###############################################################################
+#
+# Programs that can share a queue with rpc.ldmd server, these programs
+# are started by the "exec" command and are in the same process group.
+#
+EXEC	"pqact -e -k R -s e /awips2/ldm/etc/pqact.conf.priority"
+EXEC	"edexBridge -vl /awips2/ldm/logs/edexBridge-priority.log -s localhost --key-proj-id R --sem-proj-id e"
+
+#EXEC	"pqact -e -k S -s f /awips2/ldm/etc/pqact.conf"
+#EXEC	"edexBridge -vl /awips2/ldm/logs/edexBridge-std.log -s cpsbn1-oma.oma.us.ray.com --key-proj-id S --sem-proj-id f"
+#
+# rtstats uses the LDM to send product latency statistics to the UPC.
+# For more info on rtstats please see the man page.
+EXEC	"rtstats -h rtstats.unidata.ucar.edu"
+#EXEC	"pqsurf"
+#
+###############################################################################
+# Request Entries
+###############################################################################
+#
+# Request data-products from upstream LDM-s.  The syntax is
+#
+#	REQUEST	<feedset> <pattern> <host>[:<port>]
+#
+# where:
+#	<feedset>	Is the union of feedtypes to request.
+#	<pattern>	Is an extended regular-expression for the product-
+#			identifiers to match.
+#	<host>		Is the host identifier: either a hostname or IP
+#			address in "dotted-quad" format.
+#	<port>		Is the (optional) port on <host> to which to connect
+#			(the square brackets denote an option and should be
+#			omitted).
+#
+# If the same feedtype and pattern is requested from multiple hosts, then
+# the host of the first such request will be the initial primary source
+# of data-products (i.e., data-products will be rapidly sent using the
+# HEREIS message) and the other hosts will initially be alternate sources of
+# data-products (i.e., data will be sent using the COMMINGSOON and BLKDATA
+# messages).  The primary host will probably change over time -- depending on
+# which host can deliver the data-products most quickly on average.
+#
+#
+# DEFAULT - Satellite Ancillary EDEX
+#
+REQUEST NIMAGE "CloudAndMoistureImagery" 10.0.0.233 # GOES Stiched Channels (Unidata)
```

Comment:
Do we want these to have what our users should be requesting from? `idd.unidata.ucar.edu`
And then we obviously modify them to our internal relays when we use them?

Reply:
oh yeah, you're right

Reply:
Okay cool, my only other question. I thought previously we had some of our public (and maybe internal as well) entries pointing at `hrrr.unidata.ucar.edu`, is that no longer the case?

Reply:
No longer the case. Mike has all of of that data going through idd now and actually requested that we request from there and not hrrr.unidata.ucar.edu

Reply:
Okay great. Ya I thought that had changed but couldn't remember. Thanks!